process/block/preprocess/basePreProcess.go

```diff
@@ -29,6 +29,12 @@ type txsHashesInfo struct {
 	receiverShardID uint32
 }
 
+type txsForBlock struct {
```

Comment:
This should have methods like get, put, increment and so on.

Reply:
when we have a safe concurrent map, then it will be changed.